prow/cmd/deck/main.go

```diff
@@ -50,16 +51,42 @@ var (
 	redirectHTTPTo = flag.String("redirect-http-to", "", "host to redirect http->https to based on x-forwarded-proto == http.")
 	// use when behind an oauth proxy
 	hiddenOnly = flag.Bool("hidden-only", false, "Show only hidden jobs. Useful for serving hidden jobs behind an oauth proxy.")
+	runLocal   = flag.Bool("run-local", false,
+		"attempt to download data endpoints from --upstream and serve a local copy of the UI only")
```

Comment:
--upstream?

Reply:
Oops. At one point I was going to do the downloads in deck too but using curl works fine for now and having a wrapper makes it easy to run against the right $PWD. Will correct this, thanks.

Reply:
fixed

Reply:
+1 on doing the downloads as part of --run-local. Please open an issue to track the change.

Reply:
opened one here: https://github.com/kubernetes/test-infra/issues/6698